src/viewer.js

```diff
@@ -175,9 +175,9 @@ export class Viewer {
 
     const {clientHeight, clientWidth} = this.el.parentElement;
 
-    this.defaultCamera.aspect = clientWidth / clientHeight;
-    this.defaultCamera.updateProjectionMatrix();
-    this.vignette.style({aspect: this.defaultCamera.aspect});
```

Comment:
Unfortunately I think it's unclear what the right result would be here. The camera embedded in the file is likely to have its own aspect ratio, in which case I'm not sure we should override it. 

_Ideally_ I'd like to keep the camera's aspect ratio, but "letterbox" the scene, with [renderer.setViewport](https://threejs.org/docs/?q=webglr#api/en/renderers/WebGLRenderer.setViewport) or something similar. Then we could render with the camera's intended aspect ratio, without distorting what the user sees.

Reply:
Oh I got what you mean.

I thought that as I modeling some project and put in the gltf viewer, I felt un comfortable to use it.
Cause it looks distorted.

I didn't thought that there are modeler set camera aspect ratio.

But just my opinion is the user will thought strange if the camera aspect is different with webpage.
:)

Reply:
Yeah, I think both are true. We shouldn't stretch the camera to fit the webpage. But we also shouldn't ignore the camera's aspect ratio. Letterboxing is the way to go here, but I haven't implemented it so far.

Reply:
Than maby I have to close this pull request until Letterboxing (The check box for the user whether change camera aspect ratio as I understand ) implemented.

Or I can add Check box for that.

Reply:
Yeah sorry! I think just letterboxing is the way I'd prefer to go.